base/poco/Foundation/include/Poco/Logger.h

```diff
@@ -937,7 +937,7 @@ class Foundation_API Logger : public Channel
 
 protected:
     Logger(const std::string & name, Channel * pChannel, int level);
-    ~Logger();
```

Comment:
I believe that the end goal will be to remove the poco logger completely? (including the code itself)

Reply:
We could do that, but the main goal is to try removing from ClickHouse server/Keeper
Other utilities could use it if they prefer it, e.g. `clickhouse-disks`.